scss/base/4-elements/_links.scss

```diff
@@ -1,17 +1,21 @@
 a,
 .hyperlink {
-    color: $cads-palette__black;
+    color: $cads-palette__heritage-blue;
     line-height: 1.25em;
 
     &:visited {
-        color: green;
+        color: #53388D;
```

Comment:
No hex values, use one in the language/palette and if doesn't exist then we have to add it.

Reply:
Resolved in most recent commit